index.js

```diff
@@ -217,13 +219,25 @@ app.post('/submit-data', async (req, res) => {
 
     /* nää pitäis validoida ennen kantaan tallennusta */
     /* muistetaan lisätä myös CSRF-suojaus */
-    const type = req.body.type;
+    /*
+    secretario 16.1.2018
+    - muutettu "type" tietotyyppi const > var jos typecustomista tulee radio-valintojen ulkopuolinen tieto
+    - lisätty typecustom lomakkeelle, joka luetaan type-kenttään ennen syöttöä, jos siellä on dataa
+    - jos valittu muu, mutta ei syötetty dataa, kirjoitetaan kantaan "Mp"
+     */
+    var type = req.body.type;
     const content = req.body.content;
     const location = req.body.location;
     const info = req.body.info;
     const source = "ppapp";
+    const typecustom = req.body.typecustom;
     const biter = sessionInfo.name;
 
+    /* secretario 16.1.2018: checkki kun tulee typecustom radiovalintojen ulkopuolelta*/

```

Comment:
voisiko custom type tulla text-inputista, jonka nimi on myös type? Pitäisi kyllä olla mahdollista..

Reply:
testataan

Reply:
ei voi, koska tällöin saadaan sekä radiovalinnan "Mp" että annettu custom.arvo JSON:a -> kantaan leimautuu >> {"Mp","örf"}